docs/_docs/02_features/execute-terraform-commands-on-multiple-modules-at-once.md

```diff
@@ -274,6 +274,15 @@ If any of the modules fail to deploy, then Terragrunt will not attempt to deploy
 
 To check all of your dependencies and validate the code in them, you can use the `validate-all` command.
 
+To check the dependency graph you can use the `graph-dependencies` command (similar to the `terraform graph` command),
+the graph is output in DOT format The typical program that can read this format is GraphViz, but many web services are also available to read this format.
+
+     terragrunt graph-dependencies | dot -Tsvg > graph.svg
+
+In the example above it'll generate this graph
+
+![terragrunt graph-dependencies]({{site.baseurl}}/assets/img/collections/documentation/graph.png)
```

Comment:
Can you add one more sentence here that mentions that the dependency relationship is in the direction of the arrow, and that terragrunt will run the action in reverse order, bottom up?

Reply:
sure, I haven't tried this with terragrunt destroy-all and from what I read the graph is reversed

Reply:
mmm I think I should just reverse the arrow to make the graph clear